server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -92,6 +99,50 @@ public ByteSizeValue getAvailable() {
             return ByteSizeValue.ofBytes(available);
         }
 
+        public void calcMasterBaseEffectiveStage(final DiskThresholdSettings masterThresholdSettings) {
```

Comment:
naming nit/suggestion:

```suggestion
        public void setEffectiveWatermarks(final DiskThresholdSettings masterThresholdSettings) {
```